packages/commonwealth/server/controllers/server_communities_methods/put_community_stake.ts

```diff
@@ -0,0 +1,22 @@
+import { Community, UserInstance } from '@hicommonwealth/model';
+import { CommunityStakeAttributes } from '@hicommonwealth/model/build/models/community_stake';
+import { ServerCommunitiesController } from '../server_communities_controller';
+
+export type PutCommunityStakeOptions = {
+  user: UserInstance;
+  communityStake: Community.SetCommunityStakeParams &
+    Community.SetCommunityStakeBody;
+};
+
+export type PutCommunityStakeResult = CommunityStakeAttributes;
+
+export async function __putCommunityStake(
+  this: ServerCommunitiesController,
+  { communityStake }: PutCommunityStakeOptions,
+): Promise<CommunityStakeAttributes> {
+  const [newCommunityStake] = await this.models.CommunityStake.upsert(
+    communityStake,
+  );
+
+  return newCommunityStake;
```

Comment:
Should there be an authorization check to ensure only mods/admins can perform this action?

Reply:
Thanks, good catch